dist/index.html

```diff
@@ -65,10 +56,14 @@ <h2>Add your score</h2>
                 class="form-control border border-dark"
                 id="score"
                 placeholder="Add your score"
+                required
+                minlength="1"
               />
             </div>
 
-            <button type="submit" class="btn btn-outline-dark">Submit</button>
+            <button type="submit" class="btn btn-outline-dark" id="submitScore">
+              Submit
+            </button>
```

Comment:
- [ ] I tried to add a new score on the API, but I'm afraid I couldn't do that, please, the submit button should hit the POST API endpoint and post user entries. At this moment it just refreshes the page, tries to use `e.preventdefault` at the same time to avoid the default functionality for submit button, and then test if the record is saved on the API.
![image](https://user-images.githubusercontent.com/20563252/131727867-4e363993-f5d0-44ad-bd2a-e410027f72e8.png)
![image](https://user-images.githubusercontent.com/20563252/131727903-97f2d13f-25df-4027-a2a0-b1ce6d7301d9.png)

Reply:
thanks. I've fixed it